common/src/main/java/com/microsoft/identity/common/adal/internal/util/HashMapExtensions.java

```diff
@@ -133,6 +134,26 @@ private HashMapExtensions() {
         return response;
     }
 
+    /**
+     * get key value pairs from response.
+     *
+     * @param webResponse {@link HttpResponse} to convert to a map
+     * @return HashMap
+     * @throws JSONException
+     */
+    public static HashMap<String, String> getJsonResponse(HttpResponse webResponse) throws JSONException {
+        final HashMap<String, String> response = new HashMap<>();
+        if (webResponse != null && !TextUtils.isEmpty(webResponse.getBody())) {
```

Comment:
This looks like a good opportunity for code sharing with the _other_ `getJsonResponse` method inside of this class - looks like both methods could accept a `String` parameter.

[Opened an issue here](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/issues/349), since we can clean this up later.